insets/README.md

```diff
@@ -93,12 +95,13 @@ For a more complex example, see the [`EdgeToEdgeLazyColumn`](https://github.com/
 <img src="images/edge-to-edge-list.jpg" width=300>
 </a>
 
-### Animated Insets support (Experimental)
+### 🚧 Animated Insets support (Experimental)
 
 ![](images/ime-insets.gif)
 
-The library now has experimental support for [`WindowInsetsAnimations`](https://developer.android.com/reference/android/view/WindowInsetsAnimation).
-The new `imePadding()` and `navigationBarsWithImePadding()` modifiers allow content to animate with the device's on screen-keyboard (IME), while it is being animated on/off screen. This currently only works when running on devices with API 30+.
+The library now has experimental support for [`WindowInsetsAnimations`](https://developer.android.com/reference/android/view/WindowInsetsAnimation), allowing your content is react to inset animations, such as the on screen-keyboard (IME) being animated on/off screen. The `imePadding()` and `navigationBarsWithImePadding()` modifiers are available especially for this use-case. 
+
+This functionality works wherever [WindowInsetsAnimationCompat](https://developer.android.com/reference/androidx/core/view/WindowInsetsAnimationCompat) works, which at the time or writing is on devices running API 23+.
```

Comment:
Is it 23+? Haven't tested but [the impl](https://cs.android.com/androidx/platform/frameworks/support/+/androidx-master-dev:core/core/src/main/java/androidx/core/view/WindowInsetsAnimationCompat.java;l=69) looks like 21+

Reply:
Just tested, works on API 21+. Good catch 👍